pyperformance/venv.py

```diff
@@ -232,7 +232,7 @@ def _install_pyperf_optional_dependencies(self):
                 print("WARNING: failed to install %s" % req)
                 pass
 
-    def ensure_reqs(self, requirements=None):
+    def ensure_reqs(self, requirements=None, cwd=None):
```

Comment:
Instead of passing it in, we can infer `cwd` from `requirements` in the `elif hasattr(requirements, 'requirements_lockfile'):` case.  That seems appropriate for `VenvForBenchmarks`, which has the necessary context.